tests/components/mqtt/test_alarm_control_panel.py

```diff
@@ -223,6 +233,53 @@ async def test_ignore_update_state_if_unknown_via_state_topic(
     assert hass.states.get(entity_id).state == STATE_UNKNOWN
 
 
+@pytest.mark.parametrize(
+    ("hass_config", "expected_features"),
+    [
+        (
+            DEFAULT_CONFIG,
+            DEFAULT_FEATURES,
+        ),
+        (
+            help_custom_config(
+                alarm_control_panel.DOMAIN,
+                DEFAULT_CONFIG,
+                ({"supported_features": []},),
+            ),
+            0,
```

Comment:
```suggestion
            AlarmControlPanelEntityFeature(0),
```

Do we want to allow no supported feature is set?

Reply:
Ack on the `0` -> `AlarmControlPanelEntityFeature(0)` change, makes sense!

> Do we want to allow no supported feature is set?

I don't see why not. That's the default in the base component, as well as, as an example, in the template alarm panel integration.

If anything I'd argue that "disarm" should also be a feature that can be disabled, to also be able to support read-only alarm integrations - such as the case of systems with an event bus or programmable outputs. But that's not supported in the core integration, so kinda besides the point :)